bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -1,11 +1,26 @@
 ---
-# The network cidr and service cidr are set in the ovn-config configmap
+# The config file used by ovnkube is passed as a configmap.
```

Comment:
"The ovnkube config file. Used by both node and master processes."

Reply:
changed